### DISCUSSION TITLE
Documentation only showing Dart snippets?

### DISCUSSION THREAD

**Discussion by pranavbadami** [supabase/supabase#3490] [Questions] [answered]
Hey everyone! Really quick question here; not sure if it's just me, but the docs are only displaying snippets in Dart, not in Javascript. Any fixes here or is the bug in the documentation repo? Thank you!

![image](https://user-images.githubusercontent.com/6540790/136856543-57a326c0-1c77-4c58-a242-62273c816ac3.png)

**Comment by soedirgo** [comment, marked as answer]
Hey @pranavbadami , there's a JS section and a Dart section in the sidebar.

**Reply by soedirgo** [reply]
Not sure if this was intended though, there used to be just a language selection above the code snippets. @kiwicopple?

**Reply by pranavbadami** [reply]
Thank you! Knew I was missing something silly lol but yes I also remember the language selector above the snippets, which is why I was confused.
